ops-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.5"
+networks:
+  docker-practice-net:
+    driver: bridge
+services:
+  app:
+    image: app
+    container_name: app
+    restart: always
+    networks:
+      - docker-practice-net
+    ports:
+      - 8080:8080
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/school
+    volumes:
+      - .. ./tmp:/docker-practice-java-develop/target/*.jar
```

Comment:
what about hot reload, are you sure that ./tmp folder is enough? How should a service reload if you change the code of your api?